docs/releases/2.0.txt

```diff
@@ -282,6 +282,9 @@ Miscellaneous
   instead of dotted Python path strings. Django favors callable references
   since they provide better performance and debugging experience.
 
+* Calling `django.setup()` during a `django.setup()` call (reentrant call)
```

Comment:
Sorry, I didn't communicate clearly that I don't think a release note is needed. That's not really a  "feature" that I'd expect anyone to care about.